patroni/ha.py

```diff
@@ -257,9 +271,15 @@ def manual_failover_process_no_leader(self):
         return self._is_healthiest_node(members, check_replication_lag=False)
 
     def is_healthiest_node(self):
+        if self.is_paused() and self.cluster.failover and not self.cluster.failover.scheduled_at:
```

Comment:
So the decision here is to ignore the master role of a node (without the lock) when determining where the node is healthy enough to become a leader in the case when manual failover is already in progress. Since we don't do any demotes in the paused state except from inside the process_manual_failover_from_leader (which requires having a lock), and we explicitly forbid the Postgres master to acquire a lock during the failover by saying it is not healthy (unless it is a target for promotion), ISTM that a failover will lead to having more than one master under this logic. Is there something I'm missing?